portfolio/src/main/webapp/js/script.js

```diff
@@ -93,24 +104,24 @@ async function getText() {
 
 /** Creates an <li> element containing text.
  * @param {Object} message A JSON object representing a single comment.
+ * @param {Boolean} deletable Whether user can delete comment or not.
  */
-function createListElement(message) {
+function createListElement(message, deletable) {
   const liElement = document.createElement('li');
-  liElement.className = 'message';
-  liElement.innerText = message.text;
+  liElement.innerHTML = `<b>${message.useremail}:  </b> ${message.text}`;
```

Comment:
`innerText` if possible here too. While user email should be relatively safe, do you actually know all the valid characters in an email (and for invalid characters, what happens to them)? If not, it's better to err on the side of caution.